crates/bevy_picking/src/pointer.rs

```diff
@@ -270,6 +262,8 @@ pub enum PointerAction {
 pub struct PointerInput {
     /// The id of the pointer.
     pub pointer_id: PointerId,
+    /// The pointer entity
+    pub pointer_entity: Option<Entity>,
```

Comment:
Picking is already inconsistent enough across backends, I would strongly prefer if we didn't add more inconsistency, though I appreciate your reasoning.